### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Alljobs_Screenreader:Voice over is not announcing Header Name and level.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5821]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS2.4.6; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name: Voice Over
**MAS Reference: MAS1.3.1, MAS 2.4.6**
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the connected server->Select the database->Navigate to the SQL agent tab.
4.Go to the Operators tab->click on New operator button->Navigate to the Notification tab.
5.Start Voice over
6.Observed that Voice over is not announcing the header name.
**Actual:**
Voice Over is not announcing header name and level in notification Tab.
![image](https://user-images.githubusercontent.com/39204909/58777853-e52a2a80-85ed-11e9-9794-5b5bc3904b95.png)

**Similar issue is found under the below pages** 

1. **CSVSelectDestinationDatbse** -Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files
2.**CSVPreviewData**-Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" butto
3.**CSVModifyColumns**-Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.
4.**CSVSummary**-Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->click on next button.

5.**Summary** - Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button->Summary screen should open->Click on next button.

6.**MapDataSrcObjects**- Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button
7.**Summary**- Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button->Summary screen should open->Click on next button.
**Expected:**
Voice Over should announce the header name and level for notification tab.
**User Impact:**
User with visual and motor disabilities will find difficult to differentiate between header and the rest text and the purpose for the page.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
**MAS1.3.1, MAS2.4.6**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}
**Attachment for Reference:**

[5821_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3245954/5821_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
